log.go

```diff
@@ -72,16 +76,16 @@ func initFileLog(cfg *FileLogConfig) (*lumberjack.Logger, error) {
 	}, nil
 }
 
-func newStdLogger() *zap.Logger {
+func newStdLogger() (*zap.Logger, *ZapRecord) {
 	conf := &Config{Level: "info", File: FileLogConfig{}}
-	lg, _, _ := InitLogger(conf)
-	return lg
+	lg, r, _ := InitLogger(conf)
+	return lg, r
 }
 
 var (
-	_globalMu sync.RWMutex
-	_globalL  = newStdLogger()
-	_globalS  = _globalL.Sugar()
+	_globalMu          sync.RWMutex
+	_globalL, _globalR = newStdLogger()
+	_globalS           = _globalL.Sugar()
```

Comment:
When do we need to use `sugar`?

Reply:
I'm not sure in our situation but keep supporting it.